python/docs/conf.py

```diff
@@ -34,7 +34,6 @@
     'sphinx.ext.intersphinx',
     'sphinx.ext.imgmath',
     'sphinx.ext.autosectionlabel',
-    'm2r'
```

Comment:
You can also remove it from python/requirements-dev. Then it should be totally gone.